### PR TITLE
User/put

### PR DESCRIPTION
**Pull request by hewonjeong**
- add new function for put user : UserManager.modifyUser(userKey, userObject, callback)

### CODE REVIEW

File: src/server/users/models.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export default class UserManager {
+  static modifyUser(userKey, UserObject, cb) {
+    return new Promise((resolve, reject) => {
+      db.get(userKey, (err) => {
+        return (err) ? reject(err) : resolve();
+      });
+    }).then(()=>{
+      return new Promise((resolve, reject) => {
+        db.put(userKey, UserObject, (err) => {
```
Comment (ovekyc) [inline src/server/users/models.js]: userObject

File: src/server/users/models.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import db from '../database';
+
+export default class UserManager {
+  static modifyUser(userKey, UserObject, cb) {
```
Comment (ovekyc) [inline src/server/users/models.js]: userObject
Or you can just use key, value as a variable name.
bcz UserManager is definitely for User entity and this function also deals only User entiry

File: src/server/users/models.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          return (err) ? reject(err) : resolve();
+        });
+      });
+    }).then(()=>{
+      return cb(null);
+    }).catch((err)=>{
+      if (err.NotFound) {
+        return new Error('Not found : invalid key');
+      }
+      return new Error('Error while putting in DB');
```
Comment (ovekyc) [inline src/server/users/models.js]: You cant ensure the error is for operation get or put

File: src/test/node/users/models.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    return new Promise((resolve, reject) => {
+      UserManager.modifyUser(mockUser.key, modifedUser, (err) => {
+        return (err) ? reject(err) : resolve();
+      });
+    });
+  }).then(()=>{
+    return new Promise((resolve, reject) => {
+      testDB.get(mockUser.key, (err, value) => {
+        if (err) {
+          reject(err);
```
Comment (ovekyc) [inline src/test/node/users/models.js:91]: return

File: src/server/users/models.js (hunk 2 above)
Comment (hewonjeong) [reply src/server/users/models.js]: I think so. thx

**Comment by ovekyc** [comment]
LGTM

**Comment by festinalente91** [comment]
LGTM

**Comment by HewonJeong** [comment]
lgtm

**Comment by hewonjeong** [comment]
lgtm

**Pull request merged**